ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -1601,6 +1601,17 @@ static const func_info_t func_infos[] = {
 	F0("imagesetinterpolation",					MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_TRUE),
 	F1("imageresolution",						MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_TRUE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_LONG | MAY_BE_ARRAY_OF_LONG),
 
+	/* ext/spl */
+	F1("class_implements",						MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_STRING),
+	F1("class_parents",							MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_STRING),
+	F1("class_uses",							MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_STRING),
+	F0("iterator_apply",						MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_LONG),
+	F0("iterator_count",						MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_LONG),
+	F1("iterator_to_array",						MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_ANY | MAY_BE_ARRAY_OF_ANY),
```

Comment:
I'm wondering if the iterator_to_array result can contain references, e.g. when applied to generators? (Both in what is actually happening, and in what should be happening).

Reply:
It looks like I can with https://www.php.net/manual/en/language.generators.syntax.php#control-structures.yield.references

```
php > $f = function &() use (&$x) { $x = -1; yield $x; };
php > foreach ($f() as &$y) { $y = 41; }
php > var_export($x);
41
php > $arr = iterator_to_array($f());
php > $arr[0] = 3;
php > var_export($x);
3
```

Reply:
I don't see any examples of FN, FR, and FX being used, so I don't know which applies here